llvm/lib/Target/RISCV/RISCVRegisterInfo.h

```diff
@@ -112,11 +112,33 @@ struct RISCVRegisterInfo : public RISCVGenRegisterInfo {
 
   bool doesRegClassHavePseudoInitUndef(
       const TargetRegisterClass *RC) const override {
+    return isVRRegClass(RC);
+  }
+
+  bool isVRRegClass(const TargetRegisterClass *RC) const {
```

Comment:
Can these be static methods?

Reply:
in `RISCVRegisterInfo` class or a namespace?

Reply:
In RISCVRegisterInfo class is fine. Mainly suggested it because it would avoid the static_cast in onlyAllocateRVVReg.